CI/CD Testing

AndrewGable opened the pull request:
Testing CI/CD builds, please ignore 👍

shonsirsha committed:
fix: alter expensify-datepicker class

marcaaron committed:
Remove appBG from horizontalStackedAvatar

marcaaron committed:
Update styles on press/hover

marcaaron committed:
Transparent does not work here for multiple avatars. It must be appBG so that we get the correct overlap effect.

hungvu193 committed:
make image refreshable

marcaaron committed:
Clean up styles

marcaaron committed:
Revert a couple of things and clean up

dnlfrst committed:
Set default status bar color in Google Chrome on Android

Ollyws committed:
Changed sliderOnPress to run on UI thread

Ollyws committed:
Added updateImageOffset to sliderOnPress

Ollyws committed:
Added comment to siderOnPress

Commit:
clean up save button styles

dnlfrst committed:
Improve comment for clarity

dnlfrst committed:
Add missing whitespace

MariaHCD committed:
Clear request queue when signing out

Ollyws committed:
Made onPressIn inline

Co-authored-by: Sahil <thesahindia@gmail.com>

tgolen committed:
Merge branch 'main' into tgolen-optimize-optionlist

tgolen committed:
Add early return instead

grgia committed:
Add body color on web-only (fix ios errors)

tgolen committed:
Improve comment for prop

tgolen committed:
Fix memo function

luacmartins committed:
move colors to colors file

luacmartins committed:
update items on render

Commit:
better utils funciton name

Commit:
use new name

Commit:
trailing space

grgia committed:
Use new wordmark

roryabraham committed:
Install shellcheck npm wrapper as a dev dependency

roryabraham committed:
Add shellCheck script

chiragsalian committed:
using lodashOrderBy to clean up the logic here

roryabraham committed:
Add proper handling of exit code

roryabraham committed:
Add shellcheck to lint workflow

AndrewGable committed:
Merge branch 'main' into andrew-ci-test/3

AndrewGable committed:
Commit to trigger build

roryabraham committed:
Fix shellcheck linter warnings

roryabraham committed:
Add nicer output

marcochavezf committed:
add new checklist item to use ScrollView

roryabraham committed:
Add names to steps in lint workflow

srikarparsi committed:
Merge pull request #12978 from Expensify/chirag-order-improvements

LHN order improvements

marcochavezf committed:
update new checklist item

OSBotify commented:
:test_tube::test_tube: Use the links below to test this build in android and iOS. Happy testing! :test_tube::test_tube:
| android :robot:  | iOS :apple: | desktop :computer: |
| ------------- | ------------- | ------------- | 
| https://ad-hoc-expensify-cash.s3.amazonaws.com/android/13386/index.html  | https://ad-hoc-expensify-cash.s3.amazonaws.com/android/13386/index.html  | https://ad-hoc-expensify-cash.us-east-1.amazonaws.com/desktop/13386/NewExpensify.dmg |
| ![Android](https://api.qrserver.com/v1/create-qr-code/?size=200x200&data=https://ad-hoc-expensify-cash.s3.amazonaws.com/android/13386/index.html) | ![iOS](https://api.qrserver.com/v1/create-qr-code/?size=200x200&data=https://ad-hoc-expensify-cash.s3.amazonaws.com/android/13386/index.html) | ![desktop](https://ad-hoc-expensify-cash.us-east-1.amazonaws.com/desktop/13386/NewExpensify.dmg) |

NikkiWines committed:
update baseTextInput line height to prevent cutoff

shonsirsha committed:
fix: minor restyle password protection pdf form

grgia committed:
Standardize overlay color to 60% highlightBG

MariaHCD committed:
Merge pull request #13365 from Ollyws/fix-12723

Fix for: Zooming out using the slider bar in edit photo does not properly display the image

MariaHCD committed:
Clear reconnection callbacks upon signout

MariaHCD committed:
Merge remote-tracking branch 'origin/main' into maria-clear-request-queue

MariaHCD committed:
Clear reconnection callbacks upon sign out

MariaHCD committed:
Fix lint error

cristipaval committed:
Update expensify-common dependency

Gonals committed:
Merge branch 'main' into alberto-timezone

Gonals committed:
default to empty string and populate in different components

Gonals committed:
change  placeholder

Gonals committed:
style

tgolen committed:
Remove commented out code

tgolen committed:
Merge pull request #13215 from dnlfrst/fix-theme-color-modal-backdrop

Adapt status bar color to modal backdrop

mountiny committed:
Merge pull request #12759 from Expensify/amy-onyx-error-requesting-money

Only generate optimistic IOU report if no existing iouReportID

tgolen committed:
Remove mode prop that was always set to "default"

b1tjoy committed:
fix copy and paste text inconsistent on mWeb

tgolen committed:
Remove options that are no longer used

tgolen committed:
Micro-optimize function with native sorting and for loops

tgolen committed:
Micro-optimize

hungvu193 committed:
update comment

youssef-lr committed:
Fix method argument

youssef-lr committed:
Small improvement

Puneet-here committed:
navigate to workspaces on back button press

Puneet-here committed:
remove FullPageNotFoundView

luacmartins committed:
add created param to markCommentAsUnread

luacmartins committed:
Merge pull request #13393 from Expensify/maria-clear-request-queue

Clear request queue and reconnection callbacks upon sign out

luacmartins committed:
update tests

luacmartins committed:
update more tests

luacmartins committed:
add created to readNewestAction

thienlnam committed:
Merge pull request #13394 from Expensify/cristi_update-expensify-common

deetergp committed:
Merge pull request #13316 from shonsirsha/fix/date-picker-icon-not-centred

fix: alter expensify-datepicker class

luacmartins committed:
convert to array

roryabraham committed:
Merge pull request #12056 from vladamx/vladamx-proposal-11823-drag-and-drop-overlay

grgia committed:
Merge pull request #13383 from Expensify/georgia-wordmark

Branding: Use New Wordmark Logo

Luke9389 committed:
Merge pull request #13319 from Expensify/tgolen-optimize-optionlist

Cleanup props and callbacks for OptionsList

Luke9389 committed:
Merge pull request #13181 from hungvu193/fix-11283

Fix green outline at checkbox

luacmartins committed:
subtract 1 millisecond

roryabraham committed:
Merge pull request #13128 from Expensify/flo_lastRead

luacmartins committed:
add comment

grgia committed:
Use querySelector, update comment

grgia committed:
More descriptive comment

luacmartins committed:
update comment

marcaaron committed:
Merge pull request #13278 from Expensify/georgia-dark-fix

Change background color for mWeb scrolling

marcaaron committed:
Merge pull request #13351 from Expensify/marcaaron-fixBorder

Ensure correct background is present for `horizontalStackedAvatar` style

luacmartins committed:
rename test vars

luacmartins committed:
Merge branch 'main' into cmartins-addCreated

roryabraham committed:
Add husky to ignored directories

francoisl committed:
Revert "Display banner to enable 2FA when setting up VBBA"

aldo-expensify committed:
Handle error rendering

roryabraham committed:
Make DIRECTORIES_TO_IGNORE into an array

tgolen committed:
Merge branch 'main' into tgolen-analyze-performance-optionslist

grgia committed:
Rename screenBlur to match use, tweak overlay for avatars

grgia committed:
Merge branch 'main' into georgia-overlay

aldo-expensify committed:
Translate failed to load PDF error

luacmartins committed:
fix tests

luacmartins committed:
use db time

chiragsalian committed:
Merge pull request #13407 from Expensify/revert-12424-monil-VBA2FASetup

Revert "Display banner to enable 2FA when setting up VBBA"

roryabraham committed:
Fix word splitting

OSBotify committed:
Update version to 1.2.36-3

OSBotify committed:
Merge pull request #13410 from Expensify/version-BUILD-3569CF08-8743-4F9E-B6F6-7487EC19D4C6

Update version to 1.2.36-3 on main

roryabraham committed:
add more examples to join_by_string

marcaaron committed:
Merge pull request #13387 from Expensify/Rory-ShellCheck

[No QA] Lint bash scripts with shellcheck

tgolen committed:
Refactor the function for filtering the currency list

tgolen committed:
Rename method

tgolen committed:
Remove unnecessary method

tgolen committed:
Remove unused import

aldo-expensify committed:
Handle failure to load PDF in IOS and Android

aldo-expensify committed:
Remove unused import

marcochavezf committed:
Merge pull request #13272 from Expensify/maddylewis-patch-1

Rename docs/articles/send-money/Everything-About-Chat.md to docs/arti…

PauloGasparSv committed:
Merge pull request #13389 from Expensify/nikki-ios-line-height

Update baseTextInput line height to prevent cutoff

francoisl committed:
Merge pull request #13412 from Expensify/tgolen-fix-currency-selector

marcaaron committed:
Fix incorporation date error

OSBotify committed:
Update version to 1.2.36-4

OSBotify committed:
Merge pull request #13421 from Expensify/version-BUILD-5E71025F-37A7-4D23-8D64-D4EC52F6ADAC

Update version to 1.2.36-4 on main

youssef-lr committed:
Remove unneeded code

aldo-expensify committed:
Higher initial number to render

marcaaron committed:
Merge pull request #13230 from Expensify/revert-12124-fix/11584-web-jump-to-unread-message

Revert "fix: 11584 web jump to unread message"

OSBotify committed:
Update version to 1.2.37-0

OSBotify committed:
Merge pull request #13428 from Expensify/version-PATCH-E3951223-FFEE-437A-9D66-94619258C9A7

Update version to 1.2.37-0 on main

Commit:
Merge branch 'main' of https://github.com/0xmiroslav/Expensify into issue-11856

youssef-lr committed:
Fix IOU Preview still showing after failed pay request

grgia committed:
Use same overlay for all

grgia committed:
Use variable for overlay opacity for quicker changes. +Podfile update

b1tjoy committed:
convert setHTMLSync to normal function and move it to index.js

staszekscp committed:
add hash branch name action

Gonals committed:
conflicts

staszekscp committed:
improve hashBranchName

Gonals committed:
Merge branch 'main' into alberto-nameLength

staszekscp committed:
remove unnecessary semicolons

staszekscp committed:
fin linter issues

staszekscp committed:
fixx indentation

Gonals committed:
add maxlength

puneetlath committed:
Merge pull request #13325 from Expensify/alberto-nameLength

Reduce the room name length const

arosiclair committed:
Merge pull request #13402 from Puneet-here/fix-13276

navigate to workspaces page on back button press

luacmartins committed:
resolve conflicts

Puneet-here committed:
remove /

Puneet-here committed:
add character limit

tgolen committed:
DRY up checking the report chatType and fix a console error

tgolen committed:
DRY up checking the report chatType and fix a console error

tgolen committed:
Merge remote-tracking branch 'origin/tgolen-analyze-performance-optionslist' into tgolen-analyze-performance-optionslist

tgolen committed:
Merge remote-tracking branch 'origin/tgolen-analyze-performance-optionslist' into tgolen-analyze-performance-optionslist

tgolen committed:
Merge remote-tracking branch 'origin/tgolen-analyze-performance-optionslist' into tgolen-analyze-performance-optionslist

marcaaron committed:
Merge pull request #12505 from 0xmiroslav/issue-11856

fix composer hidden while loading messages

marcaaron committed:
Use else if

youssef-lr committed:
Use SET in sendMoney to save the iouReport

Co-authored-by: Vit Horacek <36083550+mountiny@users.noreply.github.com>

luacmartins committed:
Merge pull request #13404 from Expensify/cmartins-addCreated

Add created to MarkAsUnread and ReadNewestAction

marcochavezf committed:
Merge pull request #13390 from shonsirsha/fix/password-protection-pdf-form-restyle

fix: minor restyle password protection pdf form

marcaaron committed:
Merge pull request #13342 from Expensify/dangrous-updatecardmenuitemshape

Update presentation of Card Menu Items

luacmartins committed:
rm comment

danieldoglas committed:
Merge pull request #13420 from Expensify/marcaaron-fixFutureDate

Fix incorporation date error

Commit:
Merge pull request #13214 from Expensify/luke-fix-border-radii

Fix border radii

luacmartins committed:
Merge pull request #12739 from Expensify/youssef_refactor_payiou

Refactor `payIOU`

grgia committed:
Make '+X' workspace text 100% opacity

marcaaron committed:
Merge pull request #13375 from Expensify/cmartins-fixPickerUpdate

Update picker items

aldo-expensify committed:
Uppercase PDF acronym following style guideline

deetergp committed:
Merge pull request #13398 from Expensify/tgolen-analyze-performance-optionslist

Improve performance of optionslist

amyevans committed:
Merge pull request #13425 from Expensify/aldo_initial-number-render-options

Higher initial number to render

Commit:
fix "Upload photo" popup menu not hiding

aldo-expensify committed:
Merge pull request #13409 from Expensify/aldo_fix-pdf-fail-text

Fix failed to load PDF error color and translation

youssef-lr committed:
Merge pull request #13388 from Expensify/marco-addNewChecklistItem

[No QA] Add new checklist item to use ScrollView in new page components

yuwenmemon committed:
Merge pull request #12808 from hungvu193/fix-12712

Fix: Offline default avatar doesn't update after connecting to the internet

Commit:
update comment for stopPropagation in file input

marcaaron committed:
Merge pull request #13448 from 0xmiroslav/issue-13435

fix "Upload photo" popup menu not hiding

OSBotify committed:
Update version to 1.2.37-1

OSBotify committed:
Merge pull request #13468 from Expensify/version-BUILD-AF1736AF-DF7B-4EC4-8879-2D91E7D45D25

Update version to 1.2.37-1 on main

fedirjh committed:
[Fix] iouAmountText not aligned with currency text

dylanexpensify committed:
Update CONTRIBUTING.md

Update policy on reporting bonus to be clearer

Julesssss committed:
Merge pull request #13441 from Puneet-here/exp-date

Remove `/` from placeholder

b1tjoy committed:
add comment for empty catch

Gonals committed:
Remove extra flex from style

daraksha-dk committed:
add spanish and english copy for currency

daraksha-dk committed:
add tooltip for currency picker

daraksha-dk committed:
use existing copy

daraksha-dk committed:
remove unused copy

dangrous committed:
Merge pull request #13476 from daraksha-dk/currency-picker-button

Add tooltip for currency picker button

luacmartins committed:
rename param to createdDate

francoisl committed:
Revert "fix composer hidden while loading messages"

b1tjoy committed:
move comment inside catch block

pecanoro committed:
Merge pull request #13396 from b1tjoy/b1tjoy-fix-10262

Fix copy and paste text inconsistent on mWeb

luacmartins committed:
fix dates

luacmartins committed:
rm unused import

AndrewGable committed:
Merge pull request #13482 from Expensify/revert-12505-issue-11856

Revert "fix composer hidden while loading messages"

francoisl committed:
Merge pull request #13475 from Expensify/alberto-closedStyle

OSBotify committed:
Update version to 1.2.37-2

OSBotify committed:
Merge pull request #13486 from Expensify/version-BUILD-0041C992-1B03-4B6F-81BB-372A234AE089

Update version to 1.2.37-2 on main

OSBotify committed:
Update version to 1.2.38-0

OSBotify committed:
Merge pull request #13495 from Expensify/version-PATCH-8C033EA1-DD33-4E3B-8DD6-CEBB5D847A1E

Update version to 1.2.38-0 on main

aldo-expensify committed:
Use measureLayout to calculate scrolling

aldo-expensify committed:
Call measureLayout safely

aldo-expensify committed:
Merge branch 'main' of github.com:Expensify/App into aldo_fix-scrolling-and-centering-regression

francoisl committed:
Prevent commit messages that end in `\` from breaking `getMergeLogsAsJSON()`

francoisl committed:
Recompile GH actions

puneetlath committed:
Merge pull request #13323 from Expensify/alberto-timezone

Do not display default placeholder for timezone selector

tienifr committed:
fix: issue

Gonals committed:
Clean message when the user starts typing

Gonals committed:
add comment

staszekscp committed:
switch from hash to pr input

Gonals committed:
Check hasOutstandingIOU in money requests

MonilBhavsar committed:
Merge pull request #13499 from Expensify/aldo_fix-scrolling-and-centering-regression

Use measureLayout to calculate scrolling

staszekscp committed:
fix linter

staszekscp committed:
remove last line

staszekscp committed:
add exit

grgia committed:
Use overlay variable

grgia committed:
Revert Podfile commit

grgia committed:
Revert podfile changes

neil-marcellini committed:
Merge pull request #13517 from tienifr/fix/13475-Web-Changing-room-name-offline-doesnt-reflect-in-LHN-until-another-chat/room-opened

fix/13475: Web - Changing room name offline doesn't reflect in LHN until another chat/room opened

youssef-lr committed:
Merge pull request #13522 from Expensify/alberto-setError

Check hasOutstandingIOU in money requests

aldo-expensify committed:
Flatten styles to avoid crash

OSBotify committed:
Update version to 1.2.38-1

OSBotify committed:
Merge pull request #13526 from Expensify/version-BUILD-9328A369-4236-4FC0-B86B-EE2C663B2A90

Update version to 1.2.38-1 on main

daraksha-dk committed:
fix button position

aldo-expensify committed:
Report can be a {}, improve check for emptyness/falsey

situchan committed:
fix "The error text message extends beyond the screen"

Gonals committed:
Merge branch 'main' into alberto-moreClosed

Gonals committed:
use existing function

youssef-lr committed:
Merge pull request #13519 from Expensify/alberto-moreClosed

Stop displaying Closed Account message when the user starts typing

OSBotify committed:
Update version to 1.2.38-2

OSBotify committed:
Merge pull request #13537 from Expensify/version-BUILD-CEF699AA-5914-4662-A96C-5ED46313CF96

Update version to 1.2.38-2 on main

chiragsalian committed:
Merge pull request #13530 from daraksha-dk/fixButtonPosition

fix button position

aldo-expensify committed:
Use StyleUtils.combineStyles because they are props

These calculated styles are passed down to other
components in our code base, we should keep them
as array.

OSBotify committed:
Update version to 1.2.38-3

OSBotify committed:
Merge pull request #13541 from Expensify/version-BUILD-AF3CA9B2-0939-44D4-9EE5-9CD27DBBC3F4

Update version to 1.2.38-3 on main

MonilBhavsar committed:
Merge pull request #13531 from Expensify/aldo_avoid-crash-empty-report

Report can be a {}, improve check for emptyness/falsey

OSBotify committed:
Update version to 1.2.38-4

OSBotify committed:
Merge pull request #13544 from Expensify/version-BUILD-CD6B22E8-5650-4760-9D79-5F57EDDC9DF4

Update version to 1.2.38-4 on main

chiragsalian committed:
Merge pull request #13483 from Expensify/cmartins-useCreatedDate

Rename created param to createdDate

tienifr committed:
fix: issue

pecanoro committed:
Merge pull request #13535 from situchan/fix/13505-error-beyond-screen

fix error message style issue on OfflineWithFeedback

OSBotify committed:
Update version to 1.2.38-5

OSBotify committed:
Merge pull request #13553 from Expensify/version-BUILD-1961A86E-9435-4399-854D-1C61F588B291

Update version to 1.2.38-5 on main

staszekscp committed:
improve logic

MariaHCD committed:
Merge pull request #13470 from Expensify/dylanexpensify-patch-1

[No QA] Update CONTRIBUTING.md

PauloGasparSv committed:
Merge pull request #13471 from fedirjh/Fix-13309

[Fix] iouAmountText not aligned with currency text

Julesssss committed:
Merge pull request #13552 from tienifr/fix/13462-Offline-created-Split-Bill-Group-is-not-displayed-among-top-chats-in-LHN

fix/13462: LHN - Offline created Split Bill Group is not displayed among top chats in LHN

staszekscp committed:
add action to remove previous links

staszekscp committed:
fix variable name

staszekscp committed:
switch to BOTIFY token

Gonals committed:
Merge pull request #13408 from Expensify/georgia-overlay

[Dark Mode] Standardize Overlay Colors

mountiny committed:
Merge pull request #13500 from Expensify/francois-fixGetMergeLogsAsJSON

Prevent commit messages that end in `\` from breaking `getMergeLogsAsJSON()`

stitesExpensify committed:
Lint fixes

stitesExpensify committed:
Fix lint

yuwenmemon committed:
Merge pull request #13559 from Expensify/stites-fixShellLint

Fix Shell Lint

chiragsalian committed:
Merge pull request #13525 from Expensify/aldo_fix-remove-workspace-employee

Flatten styles to avoid crash

OSBotify committed:
Update version to 1.2.38-6

OSBotify committed:
Merge pull request #13567 from Expensify/version-BUILD-F4F559A0-905E-4401-8551-1C3FC1A5904D

Update version to 1.2.38-6 on main

staszekscp committed:
Merge branch 'main' into ci/hash-branch-name

AndrewGable committed:
Merge pull request #13433 from staszekscp/ci/hash-branch-name

Fix for no PR number in workflow_dispatch

AndrewGable committed:
Merge branch 'main' into andrew-ci-test/3

OSBotify commented:
:test_tube::test_tube: Use the links below to test this build in android and iOS. Happy testing! :test_tube::test_tube:
| android :robot:  | iOS :apple: | desktop :computer: |
| ------------- | ------------- | ------------- | 
| https://ad-hoc-expensify-cash.s3.amazonaws.com/android/13386/index.html  | https://ad-hoc-expensify-cash.s3.amazonaws.com/ios/13386/index.html  | https://ad-hoc-expensify-cash.us-east-1.amazonaws.com/desktop/13386/NewExpensify.dmg |
| ![Android](https://api.qrserver.com/v1/create-qr-code/?size=200x200&data=https://ad-hoc-expensify-cash.s3.amazonaws.com/android/13386/index.html) | ![iOS](https://api.qrserver.com/v1/create-qr-code/?size=200x200&data=https://ad-hoc-expensify-cash.s3.amazonaws.com/ios/13386/index.html) | ![desktop](https://api.qrserver.com/v1/create-qr-code/?size=200x200&data=https://ad-hoc-expensify-cash.us-east-1.amazonaws.com/desktop/13386/NewExpensify.dmg) |
